app/commands/v2/put_content.rb

```diff
@@ -216,6 +217,10 @@ def base_path
         payload[:base_path]
       end
 
+      def content_with_base_path?
+        payload.has_key?(:base_path) || base_path_required?
+      end
+
       def base_path_required?
         !ContentItem::EMPTY_BASE_PATH_FORMATS.include?(payload[:format] || payload[:schema_name])
```

Comment:
Though not part of this diff but in this area it could be good to switch the ordering of this `||` to indicate format is the fallback rather than primary value